### PR TITLE
Focus-mode-docs-bash

### PR DESCRIPTION
**Pull request by imicy**
todo: resolve conflicts, approve, merge into target branch

Remaining issues:
1. [Overview of DevTools](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/overview)
    devtools-intro-menu-bar.msft.png needs to be updated
2. [Console features reference](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/overview)
    Some pics cannot be found and already marked in Excel sheet
3. [Network issues guide ](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/network/issues)
    Pics cannot be found and already marked in Excel sheet

### CODE REVIEW

**Comment by captainbrosset** [comment]
Thanks @imicy. I noticed your PR was targeting `main`, so I changed it to target `focus-mode-docs-bash`, since that is the branch we are using to collect all of the new screenshots.
This led to some conflicts though, so we'll have to resolve those before we can merge.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
png 3:

Joshua:
In this file, the list control is cutoff on the right and should be inside the window of screenshot, maybe closer to cursor:
https://github.com/MicrosoftDocs/edge-developer/blob/d5ccc347628a184bcafe3b3bae2de84c993a7175/microsoft-edge/devtools-guide-chromium/media/console-javascript-autocomplete.msft.png

Yan:

The present revised image is a conflict with the following:
https://github.com/MicrosoftDocs/edge-developer/blob/afc985ef2e1c5a7dabe0e2dfb750d07c980c5cef/microsoft-edge/devtools-guide-chromium/media/console-javascript-autocomplete.msft.png
which has Cons: uses a Docs webpage, has a date.

File: microsoft-edge/devtools-guide-chromium/about-tools.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ ms.date: 02/02/2022
 # About the list of tools
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/about-tools.md:10]: In the following files, should the left side of red box be moved to the right to exclude the eye icon? 
Could fix in Paint.
https://github.com/MicrosoftDocs/edge-developer/blob/afc985ef2e1c5a7dabe0e2dfb750d07c980c5cef/microsoft-edge/devtools-guide-chromium/media/console-filter-regex.msft.png
https://github.com/MicrosoftDocs/edge-developer/blob/afc985ef2e1c5a7dabe0e2dfb750d07c980c5cef/microsoft-edge/devtools-guide-chromium/media/console-filter-text.msft.png

**Comment by captainbrosset** [comment]
I resolved the conflicts by duplicating the images that were used in 2 different articles. We don't want to share images among articles, as it makes maintaining the articles harder and sometimes leads to inconsistencies in the docs.

File: microsoft-edge/devtools-guide-chromium/about-tools.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/devtools-guide-chromium/about-tools.md:10]: Done.

**Pull request merged**